### ISSUE TITLE
Thousands and millions separators in numeric values.

### ISSUE THREAD

**Issue by Dangerous-Mind** [microsoft/fluentui#14087]
#### Describe the feature that you would like added

Many times people has to fill fields with large numeric values and want to see that they are inserting the million value instead of the thousands.
It's very difficult to read a number like 1102548,22 and people would like to see something like 1 102 548,22 while inserting the values.
Maybe I'm approaching the problem in the wrong way but I could not manage to make it to work properly.
I think that allowing the insertion of char from right to left on a MaskedTextField might do the trick.

#### What component or utility would this be added to

MaskedTextField or TextField

**Comment by paulgildea** [comment]
Thanks for reaching out to us @Dangerous-Mind 

I've added this to our backlog for review but it's low on our priority list right now because we're focused on converging components across our offerings. However, we'll take this feedback into consideration as we address components like `TextField` and `MaskedTextField` moving forward.

Thanks!

**Comment by Dangerous-Mind** [comment]
Thanks @paulgildea,

Is there any workaround to manage to make this with fluentui?

**Comment by paulgildea** [comment]
@Dangerous-Mind Unfortunately I don't have a workaround off hand - you might be able to ask the community on StackOverflow to see if folks have tried to implement your desired behavior. 

@micahgodbolt If you have any quick ideas?

**Comment by Dangerous-Mind** [comment]
> yeah, nothing built in to css or anything. MaskedTextField might do it, though that adds a lot of complixity for a small change.

I tried MaskedTextField . 
The input starts in the left side of the field (the end of the number).  People would have to write all the zero values before the actual value.

I used to set the value toLocalString() and onFocus / onBlur to trick the textfield to show the separators on the office-ui-fabric-react 6.189.2. after blur. It wasn't perfect but at least it showed the user the final value with separator.
But it stopped working on the version 7.

If I use the toLocalString while inserting the values the cursor jump to the begin of the field when it reaches a separator.

Anyway, it would be very useful to have a solution for this out of the box.
